### PR TITLE
Comment old codebase

### PR DESCRIPTION
**Pull request by anhnguyen6535**
comment on these files, mainly:
- `CanvasManager.cs`
- `PopulationController.cs`
- `PopulationManager.cs`
- `DatabaseManager.cs`

### CODE REVIEW

File: Assets/Scripts/CanvasManager.cs [hunk 1]
```diff
@@ -22,26 +22,42 @@ public class CanvasManager : MonoBehaviour
     private int countY;
     private bool drawing;
     private List<Records> recordList;
+
+    /// Setup canvas
```
Comment (khoa-islearning) [inline Assets/Scripts/CanvasManager.cs:26]: unnecessary, remove

File: Assets/Scripts/CanvasManager.cs [hunk 2]
```diff
@@ -22,26 +22,42 @@ public class CanvasManager : MonoBehaviour
...
     private void Awake()
     {
         _canvas = GetComponent<Canvas>();
         _sampleList = new List<Samples>();
         _populationId = -1;
         _prevPopulationId = -1;
+
+        /// Get _graphContainer from Unity
```
Comment (khoa-islearning) [inline Assets/Scripts/CanvasManager.cs]: *from MainScene

File: Assets/Scripts/CanvasManager.cs [hunk 3]
```diff
@@ -22,26 +22,42 @@ public class CanvasManager : MonoBehaviour
...
         maxX = (decimal) _graphContainer.sizeDelta.x;
         maxY = (decimal) _graphContainer.sizeDelta.y;
     }
 
+    /// <summary>
+    /// Check every frame if a new tree object has been clicked
+    /// If so, it destroys the current graph,
+    /// reads records of the new population from database
+    /// and starts drawing
+    /// </summary>
```
Comment (khoa-islearning) [inline Assets/Scripts/CanvasManager.cs:45]: good format

File: Assets/Scripts/CanvasManager.cs [hunk 4]
```diff
@@ -50,16 +66,19 @@ private void Update()
             Debug.Log("begin graphing");
         }
 
+        /// Start drawing
```
Comment (khoa-islearning) [inline Assets/Scripts/CanvasManager.cs]: too many comments

File: Assets/Scripts/PopulationController.cs [hunk 5]
```diff
@@ -12,16 +12,23 @@ public class PopulationController : MonoBehaviour, IPointerClickHandler
...
         _canvasManager = cm;
     }
 
      private void Awake() {
     }
+
+    /// This method will be called when clicking on a tree object
+    /// It then passes the property 'name' of the object as population id
+    /// to GetSamplesForPopulation to get a list of samples belongs to the population
+    /// The sample list is passed to Canvas Manager for drawing
```
Comment (khoa-islearning) [inline Assets/Scripts/PopulationController.cs:29]: very good 👍

**Review by khoa-islearning — APPROVED** [review]
overall very good, a few fixes would be perfect
merge when done

**Comment by anhnguyen6535** [comment]
change as requested

**Pull request merged**